### PR TITLE
Improve installation doc

### PR DESCRIPTION
**Pull request by rbovard**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
+* Fork this [GitHub repository](https://github.com/geomapfish/geomapfish_website)
 * Initialize your local repository with your GitHub fork:
 
 ```
 git init
-git remote add origin https://github.com/your_username/geomapfish_website.git
+git remote add origin https://github.com/<your_username>/geomapfish_website.git
 git fetch origin
-git merge origin/master
+git pull origin master
```
Comment (kalbermattenm) [inline README.md:24]: If you pull, you do not need to fetch... pull = fetch + merge...

File: README.md [hunk 2]
```diff
@@ -1,36 +1,37 @@
...
 git submodule update --init
 ```
 
-* Install a virtual environnement and Pelican:
+* Install a virtual environment, *Pelican* and *Markdown*:
 
 ```
 virtualenv .
 Scripts\pip install Pelican==3.6.3
+Scripts\pip install markdown
```
Comment (kalbermattenm) [inline README.md:33]: Oups... Thanks about that...

**Comment by kalbermattenm** [comment]
Could you just get rid of the `git fetch origin` line ?

Thanks a lot !

**Comment by rbovard** [comment]
Done!

**Comment by kalbermattenm** [comment]
Thanks a lot, LGTM, please merge

**Pull request merged**